cobalt/bindings/code_generator_cobalt.py

```diff
@@ -22,7 +22,12 @@
 import os
 import sys
 
-import _env  # pylint: disable=unused-import
+bindings_dir = os.path.join(
+    os.path.dirname(__file__),
+    '../../third_party/blink/Source/bindings/scripts')
```

Comment:
For consistency, here and other places I think we should be using `pardir` and `join` to create the path instead of explicit `..`s and `/`s

Reply:
I can do that, but it would leave the intent less readable IMO, and it works the same either way. Please comment if you think it's important.

Reply:
It's my preference but I don't feel strongly and wouldn't expect it to break on windows, feel free to leave it as is or change at your discretion